andela/mazus-ah-backend
# #167649375 correct email functionality

### What does this PR do?
Correct email functionality

#### Description of Task to be completed?
Correct email functionality
Change the response of verifyEmail endpoint from json objects to html tags
Adds a MarkUps.js file
Correct the email verification link

#### How should this be manually tested?
1. Fetch this branch on the terminal using
git fetch origin bg-correct-email-functionality-167649375
2. Install dependencies using npm install
3. Start the server with npm run server
4. On Postman, sign up by making a POST request to /api/v1/auth/signup.
5. Check the inbox of the email you signed up with and click the link sent.
6. Wait for the response in the browser
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167649375](https://www.pivotaltracker.com/story/show/167649375)

#### Screenshots (if appropriate)
<img width="1096" alt="Screen Shot 2019-07-31 at 9 20 10 PM" src="https://user-images.githubusercontent.com/41482184/62313798-7bad8780-b489-11e9-8071-c3d7aef69d53.png">

The view on the browser

<img width="1434" alt="Screen Shot 2019-08-01 at 4 50 27 PM" src="https://user-images.githubusercontent.com/41482184/62313775-75b7a680-b489-11e9-94c5-eaf3ea200156.png">
